intg-androidtv/apps.py

```diff
@@ -14,6 +15,12 @@
     "Ziggo": {"url": "ziggogo://"},
     "Videoland": {"url": "videoland-v2://"},
     "Steam Link": {"url": "steamlink://"},
+    "Waipu TV": {"url": "waipu://tv"},
+#    "Magenta TV": {"url": "https://www.telekom.de/magenta-tv"},
+    "Zatoo": {"url": "zatoo://zattoo.com"},
+    "Pluto TV": {"url": "https://pluto.tv/"},
+    "ARD Mediathek": {"url": "https://www.ardmediathek.de/"},
+    "ZDF Mediathek": {"url": "https://www.zdf.de/"},
```

Comment:
ZDF Mediathek launch link doesn't work on Google Chromecast (friendly name mapping works). Error: "Du hast keine App dafür". Did you test this?